### ISSUE TITLE
Using SAM for games you don't own, but played on Free Weekend

### ISSUE THREAD

**Issue by ScienceDiscoverer** [gibbed/SteamAchievementManager#183]
Its clear that using SAM for games you didn't play EVER is not a good idea. But, is is possible to implement its usage for games you did played, but don't own per se? This is really annoying and dumb that steam uses achievement stats from games you don't own but played on Free Weekend, steam still counts them for total achievement percentage stat....

**Comment by treymorris** [comment]
SAM should block you from adding a game that you don't own so that shouldn't be an issue. I know that you can earn achievements during free to play weekends which means that Steam must consider that as ownership. I would guess that during that weekend while it's still free to play you can use SAM but after that time has passed you'll get the error that you no longer own that game. It would be like if someone shared a library with you. So long as you have that family sharing enabled you're free to earn achievements etc etc. If at some point the content owner stops sharing that game/library with you then you no longer own it and would be unable to use SAM. Next free to play weekend for something I'll have to try but that's my guess.

**Comment by ScienceDiscoverer** [comment]
Yes, It seems that SAM can't really do much here. It's Valve's duty to fix this broken stupid system and disable achievements on free weekends altogether, so as the only way to get achievements is to buy a game. Until then only option is to buy game... But biggest problem is, that you can't even see your achievements in steam if you don't own game... Maybe external achive-trackers is only way to see if you played that game on free weekend...

**Comment by treymorris** [comment]
If you wanted to try opening the SAM.Manager manually you might be able to see your achievements and what-not. All you need to do is pass the SAM.Manager.exe a command line arg with the ID of the game you want to manage. Your achievements and stuff are visible on SteamDB though so that's an option if you want to just view them. I would say that if you can see them on SteamDB, then you can probably bypass the "you don't own this game" logic by launching the SAM.Manager manually and passing it the game ID.

**Comment by LeanMeanKillingMachine** [comment]
Hey @syntax-tm, mind elaborating on the bypassing the 'you don't own the game' thing? See, there's a bunch of TF2 items that require achiements in games that are all gone off of the steam store. How would I launch SAM.Manager manually, and which command would I have to use so it reads the game ID? Pretty new to coding but I've tinkered with some stuff.

**Comment by gibbed** [comment]
What syntax-tm mentions won't allow you to interact with games you don't own.

**Comment by LeanMeanKillingMachine** [comment]
> What syntax-tm mentions won't allow you to interact with games you don't own.

Yeah, figured as much. It'd be on some pretty dangerous territory otherwise,
